rspamd.service

```diff
@@ -9,9 +9,39 @@ NonBlocking=true
 ExecStart=/usr/bin/rspamd -c /etc/rspamd/rspamd.conf -f
 ExecReload=/bin/kill -HUP $MAINPID
 User=_rspamd
+ConfigurationDirectory=rspamd
 RuntimeDirectory=rspamd
 RuntimeDirectoryMode=0755
+StateDirectory=rspamd
+StateDirectoryMode=0700
+LogsDirectory=rspamd
 Restart=always
 
+AmbientCapabilities=
+CapabilityBoundingSet=
+DevicePolicy=closed
```

Comment:
Rspamd will be only able to access the following pseudo devices:

/dev/null, /dev/zero, /dev/full, /dev/random, and /dev/urandom.